### PR TITLE
Some API polishing

### PR DESCRIPTION
**Pull request by webmaster128**

### CODE REVIEW

File: packages/cosmwasm-stargate/src/signingcosmwasmclient.spec.ts [hunk 1]
```diff
@@ -25,19 +25,19 @@ const { MsgDelegate } = codec.cosmos.staking.v1beta1;
 const { Tx } = codec.cosmos.tx.v1beta1;
 
 describe("SigningCosmWasmClient", () => {
-  describe("connectWithWallet", () => {
+  describe("connectWithSigner", () => {
     it("can be constructed", async () => {
       pendingWithoutWasmd();
       const wallet = await DirectSecp256k1HdWallet.fromMnemonic(alice.mnemonic, undefined, wasmd.prefix);
```
Comment (willclarktech) [inline packages/cosmwasm-stargate/src/signingcosmwasmclient.spec.ts:31]: Do we still like `Wallet` in these class names?

**Review by willclarktech — APPROVED** [review]
Looking good. I guess the Launchpad change needs a CHANGELOG entry?

File: packages/cosmwasm-stargate/src/signingcosmwasmclient.spec.ts (hunk 1 above)
Comment (webmaster128) [reply packages/cosmwasm-stargate/src/signingcosmwasmclient.spec.ts:31]: The idea is this: signer is the interface. There are many ways to implement the signer. One way is having a wallet (term used for key storages here).

File: packages/cosmwasm-launchpad/src/signingcosmwasmclient.ts [hunk 2]
```diff
@@ -149,7 +149,7 @@ export interface PrivateSigningCosmWasmClient {
 }
 
 export class SigningCosmWasmClient extends CosmWasmClient {
-  public readonly senderAddress: string;
+  public readonly signerAddress: string;
```
Comment (webmaster128) [inline packages/cosmwasm-launchpad/src/signingcosmwasmclient.ts:152]: @willclarktech ah right, I did not notice this is public. Will add a CHANGELOG entry for this change.

**Pull request merged**